src/redist/targets/packaging/windows/clisdk/dotnet.wxs

```diff
@@ -11,9 +11,9 @@
     <MajorUpgrade DowngradeErrorMessage="$(var.DowngradeErrorMessage)" Schedule="afterInstallInitialize"/>
 
     <?if $(var.Platform)=x86?>
-      <MediaTemplate CabinetTemplate="dnetx86-{0}.cab" CompressionLevel="high" />
+      <MediaTemplate CabinetTemplate="dx86-{0}.cab" CompressionLevel="high" />
```

Comment:
I wouldn't worry about these. Remember, MSIs date back to Win 3.x :)

Reply:
@joeloff
Okay - thanks for the advise.